README.markdown

```diff
@@ -547,6 +553,7 @@ if let textContainer = self.textContainer {
   // do many things with textContainer
 }
 ```
+If the optional value is required to the context, use `guard let ...` and `assertionFailure` to catch the problem in debug time.
```

Comment:
And perhaps even `fatalError` when the program can't continue with a default value/return ?